lib/test.js

```diff
@@ -70,6 +70,9 @@ Object.keys(assert).forEach(function (el) {
 	};
 });
 
+Test.prototype._capt = assert._capt
+Test.prototype._expr = assert._expr
```

Comment:
@uiureo 

> To clear this up, I have to create a module to enhance prototype, not function nor object.
Would you wait some more days?

Yesterday I tried to add prototype enhancement feature to `empower`.
However, enhancing prototype does not work well with AVA.
So this workaround is needed for a while (Thank you for the appropriate inline comment).

By the way, I just released [empower 1.0.2](https://www.npmjs.com/package/empower).
This prevents `_capt` and `_expr` from being wrapped by `Object.keys(assert).forEach(function (el) {` loop.
A little progress.

To recap, the workaround 
```js
Test.prototype._capt = assert._capt
Test.prototype._expr = assert._expr
```
is still needed, even if empower is updated to 1.0.2, but it makes a little progress.

Would you update `package.json` again?